tests/std/tests/P0356R5_bind_front/test.cpp

```diff
@@ -92,13 +112,39 @@ constexpr bool test_constexpr() {
     bound1(4);
     assert(value == 234);
 
+    // Test PMFs.
+    Cat cat{"Peppermint"};
+    auto bound2 = bind_front(&Cat::noise, cat); // stores a copy
+    assert(bound2("meow") == "Peppermint says meow");
+    cat.name = "Fluffy";
+    assert(cat.noise("hiss") == "Fluffy says hiss");
+    assert(bound2("purr") == "Peppermint says purr");
+
+    auto bound3 = bind_front(&Cat::noise, &cat); // stores a pointer
+    assert(bound3("MEOW") == "Fluffy says MEOW");
+    cat.name = "Peppermint";
+    assert(bound3("PURR") == "Peppermint says PURR");
+
+    auto bound4 = bind_front(&Cat::noise, ref(cat)); // stores a reference_wrapper, uses LWG-2219

```

Comment:
I don't believe we need to mark this as "uses LWG-2219", since LWG-2219 has become part of the working draft [\[func.require\]p1.2][]

[\[func.require\]p1.2]: https://eel.is/c++draft/func.require#1.2

Reply:
That's correct. This is a pre-existing comment that's being moved, so I don't believe that it's critical to change, but I'll update this if I push any other changes.